keel-web/src/main/kotlin/com/netflix/spinnaker/keel/rest/ArtifactController.kt

```diff
@@ -35,9 +35,12 @@ class ArtifactController(
   )
   @ResponseStatus(ACCEPTED)
   fun submitArtifact(@RequestBody echoArtifactEvent: EchoArtifactEvent) {
+    log.debug("Received artifact event")
```

Comment:
should we log here which artifact event is it?

Reply:
I went back and forth on that because the payloads may be large, but I guess you're right. I'll add it.

Reply:
Done!